src/Bridge/Laravel/Console/Commands/SearchIndexLiveCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Search\Bridge\Laravel\Console\Commands;
+
+use Illuminate\Contracts\Container\Container as ContainerInterface;
+use LoyaltyCorp\Search\Interfaces\HandlerInterface;
+use LoyaltyCorp\Search\Interfaces\IndexerInterface;
+
+final class SearchIndexLiveCommand extends SearchIndexCommand
+{
+    /**
+     * @var \LoyaltyCorp\Search\Interfaces\IndexerInterface
+     */
+    private $indexer;
+
+    /**
+     * SearchIndexLiveCommand constructor.
+     *
+     * @param \Illuminate\Contracts\Container\Container $container
+     * @param \LoyaltyCorp\Search\Interfaces\IndexerInterface $indexer
+     */
+    public function __construct(ContainerInterface $container, IndexerInterface $indexer)
+    {
+        $this->description = 'Remove any indices deriving from search handlers that are unused';
+        $this->signature = 'search:index:clean';
+
+        $this->indexer = $indexer;
+
+        parent::__construct($container);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function handleSearchHandler(HandlerInterface $handler): void
+    {
+        $this->info(\sprintf('Swapping index for \'%s\'', \get_class($handler)));
+
+        $this->indexer->indexSwap($handler);
```

Comment:
I'd like this one to report what happened 'alias from x to y removed, alias from x to y2 added'

Reply:
@merk is the best way to accomplish this via using the LoggerInterface within the Indexer service?

Reply:
I think just outputting to $output is okay

Reply:
in order to output verbose information to `$output`, I'd need to be able to retrieve the relevant information from `indexSwap` - how would you recommend to achieve this without this method being coded purposed for console output?

Reply:
The service could accept the output as nullable or just return an array with a DTO of what changed